ebpf-io/ebpf.io-website
# feat: changes to summit pages

**Describe what changes this pull request brings**
This pull request changes to Summit Pages.

**Steps to test**
1. Open the [summit-2023-schedule](https://deploy-preview-468--ebpfio.netlify.app/summit-2023-schedule/), [summit-2023](https://deploy-preview-468--ebpfio.netlify.app/summit-2023/) and [Attendee Swag](https://deploy-preview-468--ebpfio.netlify.app/swag/) pages
1. Confirm that everything looks and works as expected
